types/src/aggregator.rs

```diff
@@ -164,8 +169,11 @@ fn string_to_bytes(value: Struct) -> Result<Vec<u8>, PanicError> {
         )
 }
 
-fn to_utf8_bytes(value: impl ToString) -> Vec<u8> {
-    value.to_string().into_bytes()
+fn u64_to_fixed_size_utf8_bytes(value: u64) -> Vec<u8> {
+    // Maximum u64 identifier size is 20 characters. We need a fixed size to
```

Comment:
why is it 20 characters? can this somehow change (i.e. can we assert?)

Reply:
sorry, auto-merge😅 Because u64::max is 20 chars, this is what I meant in the comment, can update if unclear. The problem is that it we need to pass it to formatting macro and I was not able to pass a const expr there, but will double-check👍